dockerfiles/scripts/install_common_deps.sh

```diff
@@ -11,7 +11,7 @@ apt-get update && apt-get install -y --no-install-recommends \
         python3-dev
 
 # Dependencies: conda
-wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.11-Linux-x86_64.sh -O ~/miniconda.sh --no-check-certificate && /bin/bash ~/miniconda.sh -b -p /opt/miniconda
+wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.7.12-Linux-x86_64.sh -O ~/miniconda.sh --no-check-certificate && /bin/bash ~/miniconda.sh -b -p /opt/miniconda
```

Comment:
thanks for the detailed explanation in the other comment.
so this means this version of miniconda is verified to work with all the other Dockerfiles?
i haven't had a chance to see what other changes were made between miniconda3-4.5.11 and 3-4.7.12
python version stayed the same?

Reply:
Python versions are both 3, but let me confirm minor version/ able to run a model with ep for the four dockerfiles dependent on the dependency script.